Demo/Controllers/RoomListViewController.swift

```diff
@@ -65,7 +65,7 @@ class RoomListViewController: UITableViewController {
     
     private func connect() {}
     
-    private func createJoinedRoomsProvider() {
+    private func createJoinedRoomsRepository() {
```

Comment:
In other places in the codebase we are using `make` instead of `create`.   I know Java tends to use `create` but I'm pretty sure `make` is preferred with Swift.

Reply:
Same comment applies in a couple of other places...

https://github.com/pusher/chatkit-swift/blob/06fd97d0aae080d9762297b45a921e1a8b28ddd5/Chatkit/Chatkit/Chatkit.swift#L162

https://github.com/pusher/chatkit-swift/blob/06fd97d0aae080d9762297b45a921e1a8b28ddd5/Chatkit/Chatkit/Chatkit.swift#L184

Reply:
I do agree, but should probably consult with someone that it's ok for us to introduce such a change to the public API.

Reply:
After the meeting with the rest of the team I have renamed all factory methods to start with `make`.

Reply:
Nice one 👍   
PS: there were a few comments missed (if you search for "Chatkit.create" you'll find them all).